features/rt/manifests/05-mc-kargs-worker-rt.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+    labels:
+        machineconfiguration.openshift.io/role: worker-rt
+    name: 10-kargs-worker-rt
+spec:
+    kernelArguments:
```

Comment:
Just in case, I'd say the kernelArguments would be better if they are quoted (as in https://github.com/openshift/machine-config-operator/blob/master/docs/MachineConfiguration.md#kernelarguments )

Reply:
done